package.json

```diff
@@ -1,20 +1,51 @@
 {
   "name": "css-frameworks-ca",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "Test",
   "main": "index.js",
   "scripts": {
     "build": "sass src/scss:dist/css",
-    "start": "sass --watch src/scss:dist/css & live-server"
+    "start": "sass --watch src/scss:dist/css & live-server",
+    "lint": "eslint src/**/*.js",
+    "prepare": "husky install",
+    "test-unit": "jest",
+    "test": "npm run test-unit && npm run test-e2e",
+    "test-e2e": "cypress open",
+    "watch": "sass --watch src/scss:dist/css & live-server"
   },
   "keywords": [],
   "author": "Noroff",
   "license": "ISC",
   "devDependencies": {
+    "@babel/core": "^7.23.2",
+    "@babel/preset-env": "^7.23.2",
+    "cypress": "^13.5.0",
+    "eslint": "^8.53.0",
+    "eslint-plugin-cypress": "^2.15.1",
+    "eslint-plugin-jest": "^27.6.0",
+    "husky": "^8.0.3",
+    "jest": "^29.2.0",
+    "jsdom": "^22.1.0",
+    "lint-staged": "^15.0.2",
     "live-server": "^1.2.2",
+    "prettier": "^3.0.3",
     "sass": "^1.54.8"
   },
   "dependencies": {
+    "bootstrap": "^5.3.2",
```

Comment:
This should not be necessary as bootstrap is a peer dependency of bootstrap-dark-5.

Reply:
removed